app/Http/Controllers/SpotController.php

```diff
@@ -171,27 +147,24 @@ public function show($id)
         if (!$spot) {
             return redirect('/spot')->with('error', 'Spot not found');
         }
-
         // spot.blade.php に $spot 変数を渡す
         return view('spot', compact('spot', 'liked', 'likesCount','favorited', 'favoritesCount', 'comments','posts'));
 
     }
-
     private function getUVIndex($lat, $lon)
     {
         $apiKey = env('OPENWEATHERMAP_API_KEY');
         $url = "http://api.openweathermap.org/data/2.5/uvi?lat={$lat}&lon={$lon}&appid={$apiKey}";
         $response = Http::get($url);
         $data = $response->json();
         // "value" キーが存在するか確認し、存在しない場合はデフォルト値（例：0）を返す
-        return $data['value'] ?? 0;
+        // "value" キーが存在するか確認し、存在しない場合はデフォルト値（例：0）を返す
+        return $data['value'] ?? 0 ?? 0;
```

Comment:
I think the previous implementaiton is correct, should be
return $data['value'] ?? 0;

Reply:
I fixed yesterday's problem.
I have created a new pull request, please review it.